project/settings.py

```diff
@@ -87,6 +87,7 @@
     },
 ]
 
+AUTH_USER_MODEL = "user.User"
```

Comment:
Ayarlar dosyasina kullanicagimiz user'in user dosyasindaki model.py 'daki User modeli oldugunu söylüyoruz